browser-extension/api-server/index.js

```diff
@@ -56,6 +58,32 @@ app.post("/preference/", async (req, res) => {
   // res.send({ ...result[0].get({ plain: true }) });
 });
 
+
+// Feedback code from here
+
+app.post("/feedback", async (req, res) => {
+  console.log("POST feedback");
+  try {
+    
+    
+
+    await feedback.create({
+      userId: req.user_id,
+      tweetText: req.tweet_text,
+      sentiment: req.tweet_sentiment,
+      confidence: req.tweet_confidence
+      
+    });
+
+    res.send({ msg: "Feedback Sent" });
+  } catch (err) {
+    res.status(501).send({ msg: "Error sending feedback" });
```

Comment:
@Bhargav-Dave 
check the output of console.log(err) before this line. that will tell you the exact error

Reply:
The error got resolved.
The api call is working fine and the data is being sent!